src/Commands/Modules/SoftModeration/EndorseCommand.cs

```diff
@@ -5,54 +5,76 @@
 
 using BrackeysBot.Services;
 using Discord.WebSocket;
+using System.Linq;
+using System.Collections.Generic;
 
 namespace BrackeysBot.Commands
 {
     public partial class SoftModerationModule : BrackeysBotModule
     {
-        [Command("endorse"), Alias("star")]
+        [Command("endorse"), Alias("star", "stars")]
         [Summary("Endorse a user and give them a star.")]
         [Remarks("endorse <user>")]
         [RequireGuru]
         public async Task EndorseUserAsync(
-            [Summary("The user to endorse.")] SocketGuildUser guildUser) 
+            [Summary("The user to endorse.")] SocketGuildUser guildUser = null) 

```

Comment:
I don't think I understand what this change accomplishes?

Reply:
Makes the parameter optional.

Reply:
Yeah I get that, but I meant the overall functionality of the method.

At the moment, when a user is tagged, he is given a star, but if not then the stars of the guru that called the command will be shown? That doesn't really make sense to me.

Reply:
Correct, which is why I added the command below here; which is for everybody to use to view own stars.

Reply:
Why not *just* make it a seperate method then? That is accessable for non-gurus aswell?
No need to link it up with the actual endorsing method.

Reply:
Because this doesn't hurt anybody and now there is only 1 method body to maintain.

Reply:
But now there are **two** functionalities in one method!